docs/release-procedure.md

```diff
@@ -0,0 +1,25 @@
+There are a variety of other projects related to dask that are often
+co-released.  We may want to check their status while releasing
+
+
+Release per project:
+
+*   Update version number in __init__.py, setup.py,
+    and conda.recipe/meta.yaml (if present).  Commit.
+
+*   Tag commit
+
+        git tag -a x.x.x -m 'Version x.x.x'
+
+*   and push to github
+
+        git push blaze master
+        git push blaze --tags
```

Comment:
`git push blaze master --tags` ?

It is weird that the tags are not "pushed" with the branch.